src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -787,6 +819,7 @@ public ICraftingLink submitJob(final IGrid g, final ICraftingJob job, final Base
         if (this.myLastLink != null && this.isBusy()
                 && this.finalOutput.isSameType(job.getOutput())
                 && this.availableStorage >= this.usedStorage + job.getByteTotal()) {
+            this.numsOfOutput += job.getOutput().getStackSize();
```

Comment:
Move this line [here](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/blob/master/src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java#L908)